Tools: Changelogger Release fails with cryptic message when pnpm's version is wrong

`tools/changelogger-release.sh` fails when pnpm's version is wrong with the following message:
 
```
> tools/changelogger-release.sh -p -r 6.4.3 plugins/crm
jq: parse error: Invalid numeric literal at line 1, column 34
```

By using some debug lines we are able to find the following error:

```
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version) Your pnpm version is incompatible with "/home/gogdzl/dsk/git/avengers/jetpack". Expected version: ^8.6.2 Got: 9.1.1 This is happening because the package's manifest has an engines.pnpm field specified. To fix this issue, install the required pnpm version globally. To install the latest version of pnpm, run "pnpm i -g pnpm". To check your pnpm version, run "pnpm -v".
```

--- a/tools/changelogger-release.sh
+++ b/tools/changelogger-release.sh
@@ -32,6 +32,9 @@ if [[ $# -eq 0 ]]; then
 	usage
 fi
 
+# Make sure Jetpack CLI works. Otherwise stuff might fail oddly later when we try to do `jetpack dependencies | jq`.
+pnpm jetpack noop >&2
+
 # Check whether it looks like a major version bump.
 #
 # 0.x -> 0.(x+1) also counts as major.

--- a/tools/check-intra-monorepo-deps.sh
+++ b/tools/check-intra-monorepo-deps.sh
@@ -76,6 +76,9 @@ while getopts ":uUvhHRan:" opt; do
 done
 shift "$(($OPTIND -1))"
 
+# Make sure Jetpack CLI works. Otherwise stuff might fail oddly later when we try to do `jetpack dependencies | jq`.
+pnpm jetpack noop >&2
+
 if ! $VERBOSE; then
 	. "$BASE/tools/includes/spin.sh"
 	function debug {

--- a/tools/cli/cliRouter.js
+++ b/tools/cli/cliRouter.js
@@ -10,6 +10,7 @@ import { docsDefine } from './commands/docs.js';
 import { draftDefine } from './commands/draft.js';
 import { generateDefine } from './commands/generate.js';
 import * as installCommand from './commands/install.js';
+import * as noopCommand from './commands/noop.js';
 import * as phanCommand from './commands/phan.js';
 import { releaseDefine } from './commands/release.js';
 import { rsyncDefine } from './commands/rsync.js';
@@ -43,6 +44,7 @@ export async function cli() {
 	argv = draftDefine( argv );
 	argv = generateDefine( argv );
 	argv.command( installCommand );
+	argv.command( noopCommand );
 	argv.command( phanCommand );
 	argv = releaseDefine( argv );
 	argv = rsyncDefine( argv );

--- a/tools/cli/commands/noop.js
+++ b/tools/cli/commands/noop.js
@@ -1,0 +1,17 @@
+export const command = 'noop';
+export const describe = false;
+
+/**
+ * Options definition for the phan subcommand.
+ *
+ * @param {object} yargs - The Yargs dependency.
+ * @returns {object} Yargs with the options defined.
+ */
+export async function builder( yargs ) {
+	return yargs;
+}
+
+/**
+ * Entry point for the CLI.
+ */
+export async function handler() {}

--- a/tools/cli/commands/noop.js
+++ b/tools/cli/commands/noop.js
@@ -2,7 +2,7 @@ export const command = 'noop';
 export const describe = false;
 
 /**
- * Options definition for the phan subcommand.
+ * Options definition for the noop subcommand.
  *
  * @param {object} yargs - The Yargs dependency.
  * @returns {object} Yargs with the options defined.

